friends/src/axiosConfig.js

```diff
@@ -0,0 +1,7 @@
+const axiosConfig = {
+    headers: {
+       authorization: localStorage.getItem('token')
```

Comment:
Remember in the future,  letter casing DOES MATTER when indicating 'authorization', most of the times it's written with a capital `A`, and that makes a difference, so if your config doesn't work in the future, make sure to play around with that first